### ISSUE TITLE
Attempted to find non-existent uuid

### ISSUE THREAD

**Issue by jonlo** [parse-community/Parse-SDK-iOS-OSX#657]
Sometimes while I'm using localdatastore, my app crashes raising this exception 

`[NSException raise:NSInternalInconsistencyException format:@"Attempted to find non-existent uuid %@.", uuid];`

I'm using parse iOS idk version 1.9.1

my crashlytics report:

![captura de pantalla 2015-12-15 a las 14 10 43](https://cloud.githubusercontent.com/assets/420486/11811427/b3a270b0-a335-11e5-9520-c9e2bf16e034.png)

I can't figure why this happens, it should be a fixed bug but it still happens.

**Comment by otusweb** [comment]
+1 on this issue. The worst part for me is that relaunching the app does not solve the issue. So at this point the user has to delete the app from their phone...

**Comment by luyi326** [comment]
+1 This seems to happen more when the app got updated

**Comment by roeef** [comment]
For my users it seems to happen almost always for users with Motorola devices

**Comment by yz** [comment]
+1

**Comment by roeef** [comment]
This seems to happen a lot more commonly since October - no code changes,
Still Mainly Motorola devices crashing :-\
any ideas?

**Comment by avalenzuelac** [comment]
I have the same problem but happens with some samsung devices :(

**Comment by flovilmart** [comment]
This is the iOS SDK, not sure why you're mentioning samsung or motorola devices ;)

**Comment by avalenzuelac** [comment]
I know bro, but its same problem :(

**Comment by flovilmart** [comment]
Is the issue open on the android SDK? Not the same code, not the same issue

**Comment by flovilmart** [comment]
@jonlo any particular context / STR to trigger that issue?

**Comment by flovilmart** [comment]
The crash will be resolved in next release 1.17.0 as a soft error.

**Comment by julianvogels** [comment]
Crashes hard on startup, repeatedly, in the iOS simulator. But hard to reproduce with a new installation. I start to think it's because I restarted the app a few times after it had hit a break point inside _applicationDidFinishLaunchingWithOptions_, which somehow messed with the Parse SDK. Maybe it expected a value to be set (that UUID?) but couldn't find it because before it was set I had restarted the app by running it again?
